installer/README.md

```diff
@@ -115,6 +115,8 @@ installer/install
 root@root1:~/delfin-demo/delfin$ installer/install
 ```
 
+#### Execute postcheck
+
```

Comment:
Add "After successful delfin installation, use command `ps -aux | grep delfin` to verify all processes of delfin are running"